themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md

```diff
@@ -525,7 +525,7 @@ Do you want to perform this update? details
 
 Select `yes` to confirm the update.
 
-You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Service.
+You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Cloud.
```

Comment:
```suggestion
You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Cloud console.
```

Reply:
@jkodroff this does not append console in the original so lets leave this as is.